app/assets/stylesheets/refreshments/_resume.scss

```diff
@@ -66,25 +69,43 @@ $default-color: #676775;
   }
 
   &-product__total {
-    flex: 20% 1 1;
+    flex: 30% 1 1;
     font-size: 14px;
     text-align: right;
   }
 
   &-total {
     color: $alt-color;
     text-transform: uppercase;
-    font-size: 18px;
+    font-size: 15px;
     line-height: 24px;
     position: relative;
     text-align: left;
     font-weight: bold;
-    padding: 20px;
+    padding: 10px 0 0 15px;
+
+    &.fee {
+      padding-top: 20px;
+      color: $fee-color;
+    }
   }
 
   &-total__value {
     position: absolute;
-    right: 20px;
-    top: 20px;
+    right: 15px;
+    top: 10px;
+
+    &.loading__value {
+      color: $alt-color-transparent;
+      text-shadow: 0 0 5px rgba(0,0,0,0.5);
```

Comment:
Expected single space after "," in a single-line function   function-comma-space-after